STUDIO/org.openl.rules.webstudio/src/org/openl/rules/rest/service/ProjectDeploymentServiceImpl.java

```diff
@@ -246,16 +246,11 @@ public List<DeploymentProjectItem> getDeploymentProjectItems(AProject project,
                     item.setStyleForMessages(UiConst.STYLE_ERROR);
                     item.setCanDeploy(false);
                 }
-            } catch (ProjectException e) {
+            } catch (ProjectException | JAXBException e) {

```

Comment:
Do not collapse to the single exception. The reasons of error is different, so user should know where an error has been happened.